File: android/expoview/src/main/java/host/exp/exponent/fcm/ExpoFcmMessagingService.java
package host.exp.exponent.fcm;


import android.annotation.SuppressLint;

import expo.modules.notifications.service.NotificationsService;

@SuppressLint("MissingFirebaseInstanceTokenRefresh")
public class ExpoFcmMessagingService extends NotificationsService {
  public ExpoFcmMessagingService() {
    super(new ExpoFirebaseMessagingDelegate());
  }
}

File: android/expoview/src/main/java/host/exp/exponent/fcm/ExpoFcmMessagingService.kt
package host.exp.exponent.fcm

import android.annotation.SuppressLint
import expo.modules.notifications.service.ExpoFirebaseMessagingService
import expo.modules.notifications.service.interfaces.FirebaseMessagingDelegate

@SuppressLint("MissingFirebaseInstanceTokenRefresh")
class ExpoFcmMessagingService : ExpoFirebaseMessagingService() {
  override val firebaseMessagingDelegate: FirebaseMessagingDelegate by lazy {
    ExpoFirebaseMessagingDelegate(this)
  }
}
